src/osd/Watch.h

```diff
@@ -227,9 +228,6 @@ class Watch {
   /// Generates context for use if watch timeout is delayed by scrub or recovery
   Context *get_delayed_cb();
 
-  /// True if currently connected
-  bool connected();
```

Comment:
isn't it 'const'?

Reply:
I misread. The non-`const` `connected()` is actually a duplicate in classical OSD. It has been removed as a clean-up. The getter that superseded it is actually `const` qualified.